app/human-detector-app/classes/Camera.tsx

```diff
@@ -0,0 +1,55 @@
+import Group from './Group';
```

Comment:
Moved Camera.tsx, Group.tsx and User.tsx because I thought it would be better organization